lib/ui/user_interface.rb

```diff
@@ -1,13 +1,15 @@
 class UserInterface
-  def initialize(io, board, token1, token2)
+  def initialize(io, board)
     @io = io
     @board = board
-    @token1 = token1
-    @token2 = token2
   end
 
   def welcome
-    @io.display_message("\nWelcome to Tic Tac Toe. :D")
+    display_message("\nWelcome to Tic Tac Toe. :D")
```

Comment:
I've grown pretty fond of those Script containers, can we move these strings over to a script?

Reply:
Should now be in the script.